config.py

```diff
@@ -1,5 +1,6 @@
 from os import environ
 
+
```

Comment:
I believe we need the lines for dotenv here so that the .env is loaded into your environment variables. Without the load_env line, the variables wont' be in your environment to be called with environ.get(). Unless this works on your machine?